packages/compass-indexes/src/modules/search-indexes.ts

```diff
@@ -375,12 +367,11 @@ export default function reducer(
       // previous list of indexes is shown to the user.
       // If fetch fails for refresh or polling, set the status to READY again.
       error:
-        state.status === SearchIndexesStatuses.FETCHING
-          ? action.error
-          : undefined,
-      status: SearchIndexesStatuses.FETCHING
-        ? SearchIndexesStatuses.ERROR
-        : SearchIndexesStatuses.READY,
+        state.status === FetchStatuses.FETCHING ? action.error : state.error,
+      status:
+        state.status === FetchStatuses.FETCHING
```

Comment:
This was actually a nasty bug I introduced in the refactor. FetchStatuses.FETCHING is always true 😄